### PR TITLE
Principal committees are in committees_authorized so they weren't showing

### PR DESCRIPTION
**Pull request by LindsayYoung**
I'm bringing Principal Committees back.

fixes 18F/openFEC#917

### CODE REVIEW

File: templates/candidates-single.html [hunk 1]
```diff
@@ -35,7 +35,7 @@ <h5 class="entity__term__label">District</h5>
             <h5 class="entity__term__label">
               <a class="term" data-term="Principal Campaign Committee">Principal Committee</a>
             </h5>
-            {% for c in committees if c.designation == 'P' %}
+            {% for c in committees_authorized if c.designation == 'P' %}
```
Comment (jmcarp) [inline templates/candidates-single.html]: To loop over all principal committees, use `for c in committee_groups['P']`. I noticed that we were looping over all committees several times, so I moved that logic into a `groupby` in `views.py`.

Comment (LindsayYoung) [inline templates/candidates-single.html]: Sounds good. I will make the change

**Comment by arowla** [comment]
This regression ought to have been caught by the test here (or at least a test like it-- tied to the principal committee link specifically)... but it seems it hasn't been run by Travis/caught by the test runner. We should make sure that this test actually does pass with this fix.

https://github.com/18F/openFEC-web-app/blob/develop/tests/selenium/single_candidate_page.py#L15

To invoke just that test:

    $ py.test --selenium tests/selenium/single_candidate_page.py::SingleCandidatePageTests::testCommitteeLink

**Comment by arowla** [comment]
Oh and I have a fix for the syntax errors in that file here: https://github.com/18F/openFEC-web-app/pull/246

**Pull request merged**